noir-projects/noir-contracts/contracts/new_key_registry_contract/src/main.nr

```diff
@@ -0,0 +1,60 @@
+contract NewKeyRegistry {
+    use dep::authwit::auth::assert_current_call_valid_authwit_public;
+
+    use dep::aztec::{
+        keys::{PublicKeys, stored_keys::StoredKeys}, state_vars::{PublicMutable, Map},
+        protocol_types::{point::Point, address::{AztecAddress, PartialAddress}}
+    };
+
+    #[aztec(storage)]
+    struct Storage {
+		current_keys: Map<AztecAddress, PublicMutable<StoredKeys>>,
+    }
+
+    impl<Context> Storage<Context> {
+        fn init(context: Context) -> Self {
+            Storage {
+                // Ideally we'd do KEY_REGISTRY_STORAGE_SLOT instead of hardcoding the 1 here, but that is currently
+                // causing compilation errors.
```

Comment:
Is there a noir issue for this?

Reply:
Yeah https://github.com/AztecProtocol/aztec-packages/issues/7829, forgot to link it, thanks.